### ISSUE TITLE
[BUG] LocalVideoEffectsFeature.isEffectSupported(BackgroundBlurEffect) returns false until it's internally initialised

### ISSUE THREAD

**Issue by Doctoror** [Azure/azure-sdk-for-android#1492]
**Describe the bug**
Call to `isEffectSupported` is time-sensitive. It returns `false` first, until it returns `true` after some time.

***Exception or Stack Trace***
N/A

**To Reproduce**
1. Create LocalVideoStream;
2. Obtain LocalVideoEffectsFeature;
3. Create BackgroundBlurEffect;
4. Invoke `LocalVideoEffectsFeature.isEffectSupported(BackgroundBlurEffect)` on a device that supports it.

***Code Snippet***
```kotlin
val localVideoEffectsFeature: LocalVideoEffectsFeature =
    localVideoStream.feature(LOCAL_VIDEO_EFFECTS)

val backgroundBlurEffect = BackgroundBlurEffect()
localVideoEffectsFeature.isEffectSupported(backgroundBlurEffect) // returns false

scope.launch {
    delay(1000L)
    localVideoEffectsFeature.isEffectSupported(backgroundBlurEffect) // returns true
}
```

**Expected behavior**
Output of `LocalVideoEffectsFeature.isEffectSupported(BackgroundBlurEffect)` does not change over time.

**Screenshots**
N/A

**Setup (please complete the following information):**
 - OS: Android
 - IDE: Android Studio
 - Version of the Library used: `com.azure.android:azure-communication-calling:2.6.0-beta.6`

**Additional context**
I also haven't found any documentation for `LocalVideoEffectsFeature` or its source code.
Should I report missing documentation separately?

**Information Checklist**
Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issuer as an incomplete report
- [x] Bug Description Added
- [x] Repro Steps Added
- [x] Setup information Added

**Comment by jsaurezlee-msft** [comment]
Hi @Doctoror , one question, is this happening when the video preview is on or off or both? In case this happens when video is off, can you try to reproduce this while the video preview is on?

**Comment by Doctoror** [comment]
@jsaurezlee-msft 
Thanks for your response.

1. This does not happen when video is on, but not immediately. For example,
```
override fun onFirstFrameRendered() {
    localVideoEffectsFeature.isEffectSupported(backgroundBlurEffect) // returns false
    coroutineScope.launch {
        delay(1000L)
        localVideoEffectsFeature.isEffectSupported(backgroundBlurEffect) // returns true
    }
}
```
2. Regarding documentation, the page you provided does not document individual methods. I expect something like a javadoc for `isEffectSupported` that provides details like when does this return valid data, etc.

**Comment by jsaurezlee-msft** [comment]
Closing, as this issue has been fixed in latest Calling SDK Android release 2.9.0-beta.1
